src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] dataToWrite = readFromFile(fromFileName);
+        writeToFile(toFileName, dataToWrite);
+    }
+
+    private int[] readFromFile(String fileName) {
+        File file = new File(fileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String readerString = reader.readLine();
+            int supplyCount = 0;
+            int buyCount = 0;
+            while (readerString != null) {
+                final String[] splittedLine = readerString.split(",");
+                final String operationType = splittedLine[0];
+                final int operatedAmount = Integer.parseInt(splittedLine[1]);
+                if (operationType.equals("supply")) {
+                    supplyCount += operatedAmount;
+                } else {
+                    buyCount += operatedAmount;
+                }
+                readerString = reader.readLine();
+            }
+            return new int[]{supplyCount, buyCount};
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + fileName, e);
+        }
+    }
 
+    private void writeToFile(String destinationFileName, int[] dataArray) {
+        File destinationfile = new File(destinationFileName);
+        int supplyCount = dataArray[0];
+        int buyCount = dataArray[1];
+        try {
+            StringBuilder stringBuilder = new StringBuilder();
+            stringBuilder.append("supply,").append(supplyCount).append(System.lineSeparator());
+            stringBuilder.append("buy,").append(buyCount).append(System.lineSeparator());
+            stringBuilder.append("result,").append(supplyCount - buyCount);
+            String resultData = stringBuilder.toString();
```

Comment:
You should put code in try which can actually throw an error that you throw. In this case All these lines cannot throw an IOException, so please move them outside of try-catch block